[2025-03-02T14:30:03.723+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-03-02T14:15:00+00:00 [queued]>
[2025-03-02T14:30:03.733+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-03-02T14:15:00+00:00 [queued]>
[2025-03-02T14:30:03.733+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-02T14:30:03.747+0000] {taskinstance.py:1350} INFO - Executing <Task(SimpleHttpOperator): get_velib_data> on 2025-03-02 14:15:00+00:00
[2025-03-02T14:30:03.785+0000] {standard_task_runner.py:57} INFO - Started process 1447 to run task
[2025-03-02T14:30:03.801+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_to_db_dag', 'get_velib_data', 'scheduled__2025-03-02T14:15:00+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpeamv86ts']
[2025-03-02T14:30:03.813+0000] {standard_task_runner.py:85} INFO - Job 580: Subtask get_velib_data
[2025-03-02T14:30:04.004+0000] {task_command.py:410} INFO - Running <TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-03-02T14:15:00+00:00 [running]> on host 684191f110a3
[2025-03-02T14:30:04.190+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_db_dag' AIRFLOW_CTX_TASK_ID='get_velib_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T14:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T14:15:00+00:00'
[2025-03-02T14:30:04.192+0000] {http.py:123} INFO - Calling HTTP method
[2025-03-02T14:30:04.221+0000] {base.py:73} INFO - Using connection ID 'velib_api_connection' for task execution.
[2025-03-02T14:30:04.454+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=api_to_db_dag, task_id=get_velib_data, execution_date=20250302T141500, start_date=20250302T143003, end_date=20250302T143004
[2025-03-02T14:30:04.482+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-02T14:30:04.520+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
